package.json

```diff
@@ -51,9 +51,10 @@
   "dependencies": {
     "babel-register": "6.9.0",
     "fs-extra": "0.30.0",
-    "imagemin": "5.2.2",
-    "imagemin-jpegoptim": "5.0.0",
+    "imagemin": "5.3.1",
+    "imagemin-jpegoptim": "5.2.0",
     "imagemin-pngquant": "5.0.0",
+    "jpegoptim-bin": "4.0.0",
```

Comment:
Why this explicit jpegoptim-bin dependency? This package is a dependency of imagemin-jpegoptim and will come with that package?